router/logging.go

```diff
@@ -33,12 +34,15 @@ func LogResponse(reqID string, r *http.Request, status int, err *ierrors.Error,
 	}
 
 	clientIP, _, _ := net.SplitHostPort(r.RemoteAddr)
+	ctxDuration := ctxTime(r.Context())
+	latency := strconv.FormatInt(int64(ctxDuration), 10)
```

Comment:
There's no need to stringify `ctxDuration`, logrus works well with numbers.

Also, this will return the duration in nanoseconds which I see a little bit redundant. I believe millisecond precision should be enough.